src/cplus_plugin/gui/constant_raster_manager_dialog.py

```diff
@@ -674,16 +715,15 @@ def _create_context(self) -> ConstantRasterContext:
             extent=extent,
             pixel_size=30.0,  # Default 30m resolution
```

Comment:
@osundwajeff , you can use for one of the pathways defined in the raster collection for consistency when it comes to the analysis.